### PR TITLE
Added possibility to add custom SOAP headers

### PR DESCRIPTION
**Pull request by Rooobaaat**
Added possibility to add custom SOAP headers

### CODE REVIEW

**Comment by mrcbnsls** [comment]
@Rooobaaat, you don't need to have a separate commit for each and every change. You're currently spamming some peoples mailbox with your commits.

File: Kernel/GenericInterface/Transport/HTTP/SOAP.pm [hunk 1]
```diff
@@ -24,9 +24,15 @@ our $ObjectManagerDisabled = 1;
 
 Kernel::GenericInterface::Transport::SOAP - GenericInterface network transport interface for HTTP::SOAP
 
+=head1 SYNOPSIS
```
Comment (mgruner) [inline Kernel/GenericInterface/Transport/HTTP/SOAP.pm]: These POD changes are not correct. We are using a new style now.

Comment (Rooobaaat) [inline Kernel/GenericInterface/Transport/HTTP/SOAP.pm]: Hi @mgruner,
i'm sorry. I'll change this asap.

**Comment by Rooobaaat** [comment]
Hi @mgruner 
as promised here's the updated SOAP.pm version

**Comment by carlosfrodriguez** [comment]
Hi @Rooobaaat,

In the mean time we implemented this: https://github.com/OTRS/otrs/commit/06f4ae3bfda3e687c82cf1e176ae82c11255cda7

Can you review if this also covers this pull request?

((enjoy))
cr

**Comment by mgruner** [comment]
I believe it covers the requested feature and therefore will close this PR. Please feel free to follow up with more information.

**Comment by mgruner** [comment]
Reopening. The mentioned commit only implements this for response headers, but not for request headers. This should be aligned.

**Pull request closed without merging**